src/intl/es.json

```diff
@@ -836,6 +831,11 @@
         "readMoreLink": "Leer más...",
         "calculatorLink": "Calculator"
       },
+      "section3": {
+        "date": "febrero 2023",
+        "title": "Complex Clipping for Improved Generalization in Machine Learning",
+        "readMoreLink": "Read more..."
+      },
```

Comment:
Out of curiosity, was wondering why section3 was moved below section6? It will no longer be in order.